requirements.txt

```diff
@@ -11,9 +13,14 @@ django-allauth==0.41.0
 django-heroku==0.3.1
 django-rest-auth==0.9.5
 djangorestframework==3.11.0
+drf-yasg==1.17.1
 gunicorn==20.0.4
 idna==2.8
 importlib-metadata==1.4.0
+inflection==0.3.1
+itypes==1.1.0
+Jinja2==2.11.1
```

Comment:
What are some of these requirements for? Like `Jinja2` for example

Reply:
or does it all just come with `drf-yasg`

Reply:
I just tried to install `drf-yasg` on a brand new virtualenv, and turns out, it does require those packages